package.json

```diff
@@ -29,16 +29,16 @@
   },
   "devDependencies": {
     "ass": "git://github.com/jrgm/ass.git#5be99ee7abc9fcf63f9ebcc37b151b9c822146d1",
```

Comment:
General question, noticed we removed `ass` from other services. Should we do a follow up issue to do same?